reporter/src/main/kotlin/reporters/ExcelReporter.kt

```diff
@@ -469,3 +470,11 @@ private fun RichTextString.limit(maxLength: Int = MAX_EXCEL_CELL_CONTENT_LENGTH)
 
 private fun String.limit(maxLength: Int = MAX_EXCEL_CELL_CONTENT_LENGTH) =
     takeIf { it.length <= maxLength } ?: "${take(maxLength - ELLIPSIS.length)}$ELLIPSIS"
+
+private val ScopeExclude.description: String get() =
+    buildString {
+        append(reason)
+        if (comment.isNotBlank()) {
+            append(" - ").append(comment)
+        }
+    }
```

Comment:
So, this is the third time we have this code block now. How about instead defining a more generic helper function (in `Utils.kt`) like

    fun joinNonBlank(vararg strings: String, separator: String = " - ") =
        strings.mapNotNull { it.takeUnless { it.isBlank() } }.joinToString(separator)

and calling it like

    joinNonBlank(exclude.reason, exclude.comment)

?